src/packages/components/HomePage/CreateElection.tsx

```diff
@@ -0,0 +1,72 @@
+import Button from "@components/Button";
```

Comment:
The whole component is aligned left. This looks weird with the outer borders you added.
Lets center the general elements positioning, and keep the align left only for the input and select in the form + labels

![Captura de Tela 2024-03-20 às 19 47 29](https://github.com/podcodar/voting-system/assets/14007745/e5ae2f19-d514-40a7-8270-5b765d8aaf1b)